src/services/db-manager.ts

```diff
@@ -81,23 +88,55 @@ export function grantPostgresRoleQuery(user: string) {
   `;
 }
 
-export function revokePostgresRoleQuery(user: string, dbId: string) {
-  return `
-    REVOKE SELECT ON ALL TABLES IN SCHEMA public FROM ${user};
-    REVOKE INSERT ON ALL TABLES IN SCHEMA public FROM ${user};
-    REVOKE UPDATE ON ALL TABLES IN SCHEMA public FROM ${user};
-    REVOKE DELETE ON ALL TABLES IN SCHEMA public FROM ${user};
-    REVOKE EXECUTE ON ALL FUNCTIONS IN SCHEMA public FROM ${user};
-    REVOKE EXECUTE ON ALL PROCEDURES IN SCHEMA public FROM ${user};
-    REVOKE USAGE, SELECT ON ALL SEQUENCES IN SCHEMA public FROM ${user};
-    REVOKE CREATE ON SCHEMA public FROM ${user};
-    REVOKE CONNECT ON DATABASE ${dbId} FROM ${user};
+export function grantPostgresGroupRoleQuery(user: string, dbId: string, versionString: string) {
+  if (['0.0.17', '0.0.18', '0.0.20', '0.0.21'].includes(versionString)) {
```

Comment:
this is repeated everywhere, and is also something that looks dynamic... so maybe you can just add it into a constant and reuse?